packages/syntax-core/src/Card/Card.module.css

```diff
@@ -0,0 +1,11 @@
+.card {
+  border-radius: 24px;
+  padding: 36px;
+  background-color: #fff;
+}
+
+@media (max-width: 384px) {
```

Comment:
@jliotta could you check with @ajmooo to see what the actual value should be?

So far we used these:
```
sm: 0 - 480px;
md: 480 - 960px;
lg: 960+ px
```

Also, can we make this [mobile first](https://zellwk.com/blog/how-to-write-mobile-first-css/)?

```css
.card {
  border-radius: 24px;
  padding: 28px;
  background-color: #fff;
}

@media (min-width: 480px) {
   .card {
     padding: 36px;
   }
}
```